### PR TITLE
[BugFix] build script supports Python3 and fix file path typos (#9563)

### LINKED ISSUE #9563: [BUGFIX] fe build failed on machine with only Python3 installed
### Steps to reproduce the behavior (Required)
For user only installed Python 3.x, when run `mvn install -DskipTests` in fe folder, he will encounter problem blew:
image

### Expected behavior (Required)
build fe successfully

### Real behavior (Required)
build failed when maven try execute a python script
### StarRocks version (Required)
 main branch

### PR DESCRIPTION
**Pull request by packy92**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #9563

## Problem Summary(Required) ：

1. add python3 support in env.sh.
2. Add a default python execute command property and update it when user set a PYTHON environment variable. Use this property in antrun-plugin to ensure build successfully in platform with Python2.x installed or PYTHON environment variable had been set. This won't help user who only installed Python3 and without setting PYTHON environment variable.

### CODE REVIEW

**Review by imay — DISMISSED** [review]
LGTM

**Comment by kangkaisen** [comment]
run starrocks_fe_unittest

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by stdpain** [comment]
in other planform won't load env.sh

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/pom.xml [hunk 1]
```diff
@@ -828,15 +828,15 @@ under the License.
                                     <arg value="--java_out=${basedir}/target/generated-sources/proto"/>
                                     <arg value="${starrocks.home}/gensrc/proto/lake_service.proto"/>
                                 </exec>
-                                <exec executable="python">
+                                <exec executable="${PYTHON}">
```
Comment (stdpain) [inline fe/fe-core/pom.xml:831]: suggest make env PYTHON as a variable

Comment (packy92) [reply fe/fe-core/pom.xml:831]: like this `<exe exec executable="${PYTHON}">` ?

Comment (stdpain) [reply fe/fe-core/pom.xml:831]: you can ref thrift.

```
<profile>
  python3 profile:
<profile>
```

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/pom.xml (hunk 1 above)
Comment (packy92) [reply fe/fe-core/pom.xml]: please review the latest commit.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by kangkaisen — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Pull request merged**